pkgs/build-support/testers/shfmt/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  shfmt,
+  stdenvNoCC,
+}:
+# See https://nixos.org/manual/nixpkgs/unstable/#tester-shfmt
+# or doc/build-helpers/testers.chapter.md
+{
+  name,
+  src,
+  indent ? 2,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  __structuredAttrs = true;
+  strictDeps = true;
+  inherit name src indent;
```

Comment:
Why not this:

```suggestion
  inherit src indent;
  name = "shfmt-${name}";
```

Reply:
Ah, I had misunderstood what you meant in https://github.com/NixOS/nixpkgs/pull/385940#discussion_r1978801329, taking it to mean the user should always add the prefix manually. Sorry about that! Fixed in the latest.